public/keynote/keynote.css

```
@@ -0,0 +1,254 @@
+  --navy-600: #3a3a48;
+  --cyan:     #fc6c00;
+  --cyan-2:   #fba40c;
+  --amber:    #fba40c;
+  --coral:    #e63c24;
+  --ink:      #30303c;
+  --paper:    #fbf7f1;
+  --paper-2:  #ece3d6;
+  --white:    #ffffff;
+
+  --font-sans: "Inter", "Noto Sans JP", -apple-system, "Hiragino Kaku Gothic ProN", "Yu Gothic", sans-serif;
+  --font-mono: "JetBrains Mono", ui-monospace, "SFMono-Regular", Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)